roles/satellite-clone/tasks/main.yml

```diff
@@ -197,19 +197,25 @@
   file:
     path: /var/opt/rh/rh-postgresql12/lib/pgsql/data/postgresql.conf
     state: absent
-  when: satellite_version in ["6.8"]
+  when: satellite_version in ["6.8", "6.9"]
 
 - name: Restore selinux context on the filesystem
   command: restorecon -R /
   when: restorecon
 
 # This file tells the candlepin puppet module in the Satellite installer that
 # candlepin is already set up. We remove it so candlepin is set up correctly.
-- name: Remove cpdb_done file
+- name: Remove legacy cpdb_done file
   file:
     path: /var/lib/candlepin/cpdb_done
     state: absent
 
+# The location of this file changed in 6.9
+- name: Remove cpdb_done file
+  file:
+    path: /var/lib/candlepin/.puppet-candlepin-cpdb-create-done
```

Comment:
You could fold this and the legacy path into a list and do a with_items.

Or put the filepath as a variable as for each satellite version there is always just one matching.

Or you leave it like this, as it doesn't hurt ;)

Reply:
I like with_items - will do